content/en/agent/kubernetes/_index.md

```diff
@@ -74,13 +74,31 @@ This chart adds the Datadog Agent to all nodes in your cluster via a DaemonSet.
 
 Next, enable the Datadog features that you'd like to use: [APM][5], [Logs][6]
 
-**Note**: For a full list of the Datadog chart's configurable parameters and their default values, refer to the [Datadog Helm repository README][7].
+**Notes**:
+
+- For a full list of the Datadog chart's configurable parameters and their default values, refer to the [Datadog Helm repository README][7].
+- If `gcr.io/datadoghq` is not accessible in the deployment region, the Datadog docker hub registry images can be used [8][9].
```

Comment:
If Google Container Registry (`gcr.io/datadoghq`) is not accessible in your deployment region, you can use Docker Hub registry by changing the following parameters in the `values.yaml` file:

Reply:
[8] & [9] are missing the text body, no?